sys/arm64/arm64/locore.S

```diff
@@ -597,7 +618,12 @@ build_l1_block_pagetable:
 
 	/* Build the L1 block entry */
 	orr	x12, x7, #L1_BLOCK
+#ifdef ATTR_DEFAULT_LOW
+	orr	x12, x12, #(ATTR_DEFAULT_LOW)
+	orr	x12, x12, #(ATTR_DEFAULT_HIGH)
+#else
 	orr	x12, x12, #(ATTR_DEFAULT)
+#endif
```

Comment:
```suggestion
	orr	x12, x12, #(ATTR_DEFAULT_LOW)
#if ATTR_DEFAULT_HIGH != 0
	orr	x12, x12, #(ATTR_DEFAULT_HIGH)
#endif
```

Reply:
I'm trying to reduce the diff to FreeBSD here so kept the non-Morello case as it is.